docs/source/links/rest.md

```diff
@@ -0,0 +1,148 @@
+
+<h3 id="export.arguments">Arguments</h3>
+
+A `@rest` directive takes three arguments:
+  - path: path to rest endpoint. This could be a path or a full url. If a path, add to the endpoint given on link creation or from the context is concatenated to it.
+  - method: the HTTP method to send the request via (i.e GET, PUT, POST)
+  - type: The GraphQL type this will return
+  - endpoint: which endpoint (if using a map of endpoints) to use for the request
+
+<h3 id="export.variables">Variables</h3>
+
+You can use query variables inside the path argument of your directive for example
+
+```graphql
+query postTitle {
+  post(id: "1") @rest(type: "Post", path: "/post/:id") {
+    id
+    title
+  }
+}
+```
+
+<h2 id="export">@export directive</h2>
+
+The export directive re-exposes a field for use in a later (nested) query. An example use-case would be getting a list of users, and hitting a different endpoint to fetch more data using the exported field in the REST query args.
+
+<h3 id="export.arguments">Arguments</h3>
+  - as: the string name to create this as a variable to be used down the selection set
+
+<h3 id="export.notes">Notes</h3>
+
+These are the same semantics that will be supported on the server, but when used in a rest link you can use the exported variables for further calls (i.e. waterfall requests from nested fields)
+
+```graphql
+const QUERY = gql`
+  query RestData($email: String!) {
+    users @rest(path: '/users/email/:email', params: { email: $email }, method: 'GET', type: 'User') {
+      id @export(as: "id")
+      firstName
+      lastName
+      friends @rest(path: '/friends/:id', params: { id: $id }, type: '[User]') {
+        firstName
+        lastName
+      }
+    }
+  }
+`;
+```
+
+<h2 id="mutation">Mutations</h2>
+
+You can write also mutations with `apollo-link-rest`, for example:
+
+```graphql
+const createPostMutation = gql`
+  fragment PublishablePostInput on REST {
+    title: String
+  }
+  mutation publishPost($input: PublishablePostInput!) {
+    publishedPost(input: $input)
+      @rest(type: "Post", path: "/posts/new", method: "POST") {
+      id
+      title
+    }
+  }
+`;
+```
+
+<h2 id="example">Example apps</h2>
+
+To get you started, here are some example apps:
+
+* [Simple](https://github.com/apollographql/apollo-link-rest/tree/master/examples/simple):
+  A very simple app with a single query.
+* [Advanced](https://github.com/apollographql/apollo-link-rest/tree/master/examples/advanced):
+  A more complex app with .
+
+If you have an example app that you'd like to be featured, please send us a PR!
+😊 We'd love to hear how you're using `apollo-link-rest`.
+
+
+<h2 id="roadmap">Roadmap</h2>
+
+- Write nested queries that help you fetch data from your existing GraphQL servers, and interleave that with other queries from your REST APIs
```

Comment:
Needs more detail or possibly an example so people know how to contribute

Reply:
In fact it works so I remove it